### PR TITLE
Nikovian Atmospherics

### PR DESCRIPTION
**Pull request by Wesreidau**
Overhaul of the Atmospherics department.

Atmos

* Reduce the size and complexity of the Atmospherics department overall.
* Refocused Atmospherics from an impromptu bombmaking and phoron-griefing shop, to an efficient life-support system.
* Added a built-in waste gas cooling radiator.
* Employed high-power and pressure regulating valves where appropriate.
* Added a distro loop heating and cooling machine as a stock feature.
* External connectors for pumps and scrubbers now have far more local gas pumps for efficient operation.
* Reworked a few turfs and air alarms.

### CODE REVIEW

**Comment by Lohikar** [comment]
Tested this briefly, noticed a few things.

First, atmos seems to spawn without the air supply being at all functional. While it's good to have something for atmos to do, air at the least should work.

Second, there's two instances of pumps/pipes overlapping scrubbers. Could you please see if you can shuffle those around a bit so they don't overlap?

Third, this area.

The function of the three lines is not clear until you remove the tiling below, please label these adapters.

**Comment by LordFowl** [comment]
As Keknar has been busy this past week I will give him until the end of this week to either release his own version, set-up a deadline for the release of his own version to be held to, or indeed to approve this version before I consider setting about to approve it myself.

**Comment by skull132** [comment]
See if he's even interested in doing anything with it, or if we can just skip that week and go with reviewing this.

**Comment by LordFowl** [comment]
I have already pinged him. As he does not frequently check the github, setting this ultimatum without actually ever telling him would've been rather devious.

**Comment by skull132** [comment]
Faaack, it conflicted. I'll sort it out tomorrow for you, Nikov. Just keep doing your thing until then.

**Comment by Wesreidau** [comment]
Addressed Lohikar's concerns, changed the file, put Atmospherics Maintenance area into the halls west of atmospherics, added air alarms that were missing.

**Comment by MarinaGryphon** [comment]
you're doing god's work son

also is there any room for expansion or improvement? i think you mentioned a second room for additions, can you show that in a screenshot?

**Comment by SierraKomodo** [comment]
Copy+pasted from my post on the new map feedback thread:

I feel like this has the setup start out too optimized. There's really nothing to change, fix, or customize as an atmos tech that I can see because any changes I would have made to the old map system are already in place, which results in less things for an atmos tech to do for 'setup' phase (If they so desired), and less difference between an individual tech's atmospherics setups (Which provided a lot of interesting IC conflict and/or discussions regarding atmos). My main complaint with the current iteration on newmap is it's too cramped to properly make any meaningful adjustments without having to tear up literally everything and re-pipe it all.

**Comment by Wesreidau** [comment]
@SierraKomodo
First, I think it is noteworthy that engineers don't radically repipe the engine on round start even though improvements can be made. Unlike atmospherics, the basic setup is "optimized" enough that individual tastes are bwoinkable concerns. The atmospheric culture of changing the pipes stems from most maps having rather lousy basic setups.

But let's assume I go through and replace all the high power or pressure regulator pumps with standard pumps and allow the big space to the east of the portable canister storage to be a playground of custom mixing apparatus. How would that be? Incidentally, these are changes already in my post feedback commit. They just aren't in the screenshot, because its old or cropped out. Or perhaps the lack of N2O recovery is noticed and they want a filter for that. Or a phoron and oxy line to the engine... it fits if you build it. If it doesn't fit, then its not optimized for the tech's tastes. Either way I think your concern is a legitimatee one that I already addressed in my first feedback pass.

**Comment by Wesreidau** [comment]
Quick note: any additional screens will have to wait until Im back from vacation. Feel free to download and check for yourself.

**Comment by LordFowl** [comment]
After speaking with Keknar, he has decided to approve this, and push his modifications only after gauging the response to these edits. Provided the merge conflict is resolved, I also approve of these edits.

File: maps/Aurora-MapDev/aurora-3.dmm [hunk 1]
```diff
@@ -83,7 +83,7 @@
 "abE" = (/obj/structure/bed/chair/office/dark,/obj/effect/floor_decal/corner/blue{dir = 10},/obj/effect/landmark/start{name = "Security Cadet"},/turf/simulated/floor/tiled,/area/security/vacantoffice2)
 "abF" = (/obj/effect/floor_decal/corner/blue{icon_state = "corner_white";dir = 9},/turf/simulated/floor/tiled,/area/security/vacantoffice2)
 "abG" = (/obj/effect/floor_decal/corner/blue{icon_state = "corner_white";dir = 8},/turf/simulated/floor/tiled,/area/security/vacantoffice2)
-"abH" = (/obj/machinery/door/airlock/glass_security{name = "Briefing Room";req_access = list(63)},/obj/machinery/door/firedoor,/turf/simulated/floor/tiled,/area/security/vacantoffice2)
+"abH" = (/obj/machinery/door/airlock/glass_security{name = "Briefing Room";req_access = list(1)},/obj/machinery/door/firedoor,/turf/simulated/floor/tiled,/area/security/vacantoffice2)
```
Comment (Alberyk) [inline maps/Aurora-MapDev/aurora-3.dmm]: Why is this being changed?

Comment (skull132) [inline maps/Aurora-MapDev/aurora-3.dmm]: Outdated now.

**Pull request merged**